//test return 42
// test division by zero after the return does not raise exception
int main()
{
  int x = 42;
  int y = 0;
  return x + y;
  int z = x / y;
  return z;
}
